pkg/constants/constants.go

```diff
@@ -239,3 +239,7 @@ const (
 	// multicluster field value for context field.
 	LogContextMulticluster = "multicluster"
 )
+
+var (
+	SupportedProtocols = []string{ProtocolHTTP, ProtocolHTTPS, ProtocolTCP, ProtocolGRPC, ProtocolTCPServerFirst}
```

Comment:
HTTPS is not a supported protocol for in-mesh traffic.

Reply:
Removed 👍🏾